packages/ui-logic/src/csv.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { parseRecipients } from "./csv";
+
+const validText = `
+tiov15nuhg3l8ma2mdmcdvgy7hme20v3xy5mkxcezea,4
```

Comment:
The initial line break should not be valid. You need to start with 

```
`tiov15nuhg3l8ma2mdmcdvgy7hme20v3xy5mkxcezea,4
```

in the first line

Reply:
Forget about this one. Our parser allows empty lines. Not strictly CSV but should do harm